lib/task-data/tasks/rm-bmc-credentials.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Remove BMC Credentials',
+    injectableName: 'Task.Remove.BMC.Credentials',
+    implementsTask: 'Task.Base.Linux.Commands',
+    options: {
+        user: [],
+        commands: [
+            {
+                command: 'sudo ./remove_bmc_credentials.sh',
+                downloadUrl: '/api/1.1/templates/remove_bmc_credentials.sh'

```

Comment:
Just found that v1.1 is hardcoded here. it will break when migrating form 1.1 to 2.0. need refactoring in near future, don't need to be included in this PR. @RackHD/corecommitters 

:+1: for me
:+1: for this PR for me.